compliance/azure/azure_long_stopped_instances/README.md

```diff
@@ -12,27 +12,14 @@ The policy leverages the Azure API to check all instances that have been stopped
 
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
 - *Stopped days* - Number of days an instance is stopped before it is added to the report
-- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
-- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
 
 ## Prerequisites
 
-- Azure Service Principal (AKA Azure Active Directory Application) with the appropriate permissions to manage resources in the target subscription
-- The following RightScale Credentials
-  - `AZURE_APPLICATION_ID`
-  - `AZURE_APPLICATION_KEY`
+This policy requires the Azure Resource Manager Credential. When applying the policy select the appropriate credentials
+from the list for your tenant. If such credential doesn't exist please contact your cloud admin to create the Credential.
 
-## Installation
-
-1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
-1. Grant the Azure AD Application access to the necessary subscription(s)
-1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
-1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
-
-## Required RightScale Roles
-
-- Cloud Management - credential_viewer
+The credential must contain the value *Azure RM* in the Provider field.
```

Comment:
@cdwilhelm whether it should be *Azure RM* or *azure_rm*.

Reply:
Thanks for mentioning this.  it was intended to have the UI format Azure RM to azure_rm, but it's not implemented.  so, yes please change to azure_rm.